### PR TITLE
Added more code

### PR DESCRIPTION
**Pull request by Lorenzo7005**
code ready okgxgxb zxgv gdx c

### CODE REVIEW

File: main.c [hunk 1]
```diff
@@ -1 +1,5 @@
 #import main.h;
+
+void main() {
+    printf("Hello World!");
```
Comment (JacopoMangiavacchi) [inline main.c]: Hey Lorenzo, I created a PR few days ago but you probably haven't seen it so I self approved.  I added a define in main.h called WORLD.  please use this define here in the printf.

**Review by JacopoMangiavacchi — COMMENTED** [review]
Please see comments inline

File: main.c (hunk 1 above)
Comment (Lorenzo7005) [reply main.c]: I'm on ir

Comment (Lorenzo7005) [reply main.c]: Done! @JacopoMangiavacchi

**Pull request merged**